packages/data-stores/src/reader/queries/use-subscription-manager-user-settings-query.ts

```diff
@@ -0,0 +1,33 @@
+import { useSelect } from '@wordpress/data';
+import { useQuery } from 'react-query';
+import { register as registerUserStore } from '../../user';
+import { fetchFromApi } from '../helpers';
+import type { SubscriptionManagerUserSettings } from '../types';
+
+// Credentials are only needed to create users
+const USER_STORE = registerUserStore( { client_id: '', client_secret: '' } );
+
+type EmailSettingsAPIResponse = {
+	settings: SubscriptionManagerUserSettings;
+};
+
+const useSubscriptionManagerUserSettingsQuery = () => {
+	const isLoggedIn = useSelect( ( select ) =>
+		select( USER_STORE ).isCurrentUserLoggedIn()
+	) as boolean;
+	return useQuery(
+		'siteSettings',
```

Comment:
I think it may be wise to include the `isLoggedIn` value in the key. I would propose the following key:
```typescript
[ 'read', 'email-settings',  isLoggedIn ]
```